src/components/emby-button/emby-button.css

```diff
@@ -145,8 +149,6 @@
 
     .paper-icon-button-light > img {
         width: 1.72em;
-        /* Can't use 100% height or it will stretch past the boundaries in safari */
```

Comment:
why remove?

Reply:
The item it refers to was commented out so I figured it was useless.